q2_differential/_stan.py

```diff
@@ -114,50 +155,422 @@ def _case_control_single(counts : np.array, case_ctrl_ids : np.array,
         return res
 
 
-def _case_control_sim(n=100, d=10, depth=50):
-    """ Simulate case-controls from Multinomial distribution
 
-    Parameters
-    ----------
-    n : int
-       Number of samples (must be divisible by 2).
-    d : int
-       Number of microbes
-    depth : int
-       Sequencing depth
+class BaseModel:
```

Comment:
This class should resolve https://github.com/gibsramen/BIRDMAn/issues/34
The good news is that it is a very minimal change - we could even keep most of the existing base class intact.